eth/main.cpp

```diff
@@ -148,6 +148,7 @@ void help()
 		<< "    --to <n>  Export only to block n (inclusive); n may be a decimal, a '0x' prefixed hash, or 'latest'.\n"
 		<< "    --only <n>  Equivalent to --export-from n --export-to n.\n"
 		<< "    --dont-check  Prevent checking some block aspects. Faster importing, but to apply only when the data is known to be valid.\n\n"
+		<< "    --download-snapshot <path> Download Parity Warp snapshot data to the specified path." << endl
```

Comment:
I spent so much time removing `endl`s.

Reply:
Sorry I'll fix this 😆 
Anyway I hope this to go away after https://github.com/ethereum/cpp-ethereum/pull/4597